meson.build

```diff
@@ -56,6 +56,8 @@ endif
 subdir('src/nanoarrow')
 incdir = include_directories('src/')
 
+_nanoarrow_src_dir = include_directories('src/nanoarrow')  # only for use by Cython
```

Comment:
Ideally this wouldn't exist at all; didn't look into how to resolve that closely though